### PR TITLE
Fixed links for TypeScript issues/documentation.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Fixed up links to point away from our old CodePlex repo.

@mhegazy, you might want to take a quick glance as well.

### CODE REVIEW

**Comment by samccone** [comment]
@DanielRosenwasser looks like there is one more :)
https://github.com/tastejs/todomvc/blob/9c7b1a25648f06e38b9781ac2c2e74a30d76a538/examples/typescript-react/readme.md

Thanks for this PR :smile:

**Comment by DanielRosenwasser** [comment]
D'oh! That's what I get for relying on GitHub's search. :smile:

File: examples/typescript-backbone/readme.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Articles and guides from the community:
 Get help from other TypeScript users:
 
 * [TypeScript on StackOverflow](http://stackoverflow.com/questions/tagged/typescript)
-* [Forums](http://typescript.codeplex.com/discussions)
+* [Forums](https://github.com/Microsoft/TypeScript/issues)
```
Comment (arthurvr) [inline examples/typescript-backbone/readme.md]: Isn't that more of an issue tracker than a forum?

**Comment by samccone** [comment]
@DanielRosenwasser if you squash and rebase off of the latest master the tests should work now :)
https://github.com/tastejs/todomvc/commit/703dcd3ca69acc6962d688916d514c16e5bec1fd

File: examples/typescript-backbone/readme.md (hunk 1 above)
Comment (DanielRosenwasser) [inline examples/typescript-backbone/readme.md]: Other than the issues, all we have is a Gitter channel. Discussions do also occur over our issue tracker. Perhaps we can patch this up at a later change?

**Comment by samccone** [comment]
Hey @DanielRosenwasser sorry about the rebase count on this one, I just pushed another fix that should "correctly" ignore all the typescript code now :) :smile: :dancer:

**Comment by samccone** [comment]
Thanks @DanielRosenwasser A+ work here

**Comment by DanielRosenwasser** [comment]
Awesome, thanks for the patience! :smile:

**Comment by samccone** [comment]
NP thanks for the work!

**Pull request merged**